README.md

```diff
@@ -29,7 +39,29 @@ gulp.task('example', function() {
 });
 ```
 
-### decompress()
+### Custom compression
+
+Files are processed with maximum compression and output files have the `.brotli` extension.
+
+```javascript
+var gulp   = require('gulp');
+var brotli = require('gulp-brotli');
+
+gulp.task('example', function() {
+  return gulp.src('path/to/input')
+    .pipe(brotli.compress({
+      extension: 'brotli',
+      brotliParams: {
```

Comment:
It is indeed a breaking change. This separation was done on purpose to distinguish between parameters for the plugin and parameters for iltorb

If `extension` is added to the `brotliParams` object it would also be passed to iltorb. This can give unexpected results in the future. Of course we can delete the property before passing the object, but that would exclude the use of a property with the same name in any future iltorb release.

I looked at `gulp-gzip` on how it's handled there. That plugin already has a number of miscellaneous properties for the plugin itself.

My first idea was the same as yours and since the odds are small that a name conflict arises, I could change this PR if you want. What do you think: should I delete the extension property before passing it to iltorb?